sqlitecluster/SQLite.cpp

```diff
@@ -277,6 +278,13 @@ void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
     _mostRecentSQLiteErrorLog = "{SQLITE} Code: "s + to_string(iErrCode) + ", Message: "s + zMsg;
     SRedactSensitiveValues(_mostRecentSQLiteErrorLog);
     SINFO(_mostRecentSQLiteErrorLog);
+
+    // This is sort of hacky to parse this from the logging info. If it works we could ask sqlite for a better interface to get this info.
+    if (SStartsWith(zMsg, "cannot commit")) {
+        // 17 is the length of "conflict at page" and the following space.
+        const char* offset = strstr(zMsg, "conflict at page") + 17;
+        _conflictPage = atol(offset);
+    }
```

Comment:
This seems pretty fragile but I guess it's fine if it works and SQLite doesn't change this out from under us before we ask for a better interface. NAB, just noting for the record.

Reply:
I agree. There's an ongoing conversation about this. If we get some other interface implemented there, we can change this to match.